packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -126,9 +126,15 @@ export const Icon: StoryObj<TagProps> = {
 export const Clickable: StoryObj<TagProps> = {
   args: {
     label: "Starship",
-    onClick: function noRefCheck(event) {
-      event;
-    },
+  },
+  play: async ({ args, canvasElement, step }) => {
+    await step("remove the tag on click", async () => {
+      const canvas = within(canvasElement);
+      const tag = canvas.getByText(args.label);
+      await userEvent.click(tag);
+      expect(args.onClick).toHaveBeenCalledTimes(1);
```

Comment:
I am not 100% sure but this should be a function added by [@storybook/addon-interactions](https://storybook.js.org/addons/@storybook/addon-interactions)
```
Note that the fn function will assign a spy to your arg, so that you can assert invocations.
```